lib/process.js

```diff
@@ -12,13 +15,14 @@ function ProcessInfo (defaults) {
   this.ppid = null
   this.root = null
   this.coverageFilename = null
-  this.nodes = [] // list of children, filled by buildProcessTree()
-
-  this._coverageMap = null
 
   for (var key in defaults) {
     this[key] = defaults[key]
   }
+
+  if (!this.uuid) {
+    this.uuid = uuid()
```

Comment:
Why not just set this at the start of the function?

Reply:
When we're instantiating a ProcessInfo from a file, it'll have a uuid in the `defaults` object.  Doing it here prevents calling `uuid()` unnecessarily.

Reply:
Ahh yes `uuid()` can be expensive.. Good call then.